metrics_statistics_msgs/msg/MetricsMessage.msg

```diff
@@ -14,9 +14,12 @@
 # Name metric measurement source, e.g., node, topic, or process name
 string measurement_source_name
 
-# Name of the metric being measured, e.g. cpu_percentage, free_memory_mb, message_age, etc
+# Name of the metric being measured, e.g. cpu_percentage, free_memory_mb, message_age, etc.
 string metrics_source
 
+# Unit of measure of the metric, e.g. percent, mb, seconds, etc.
```

Comment:
Will this be something other packages will specify/use? My first worry is that these units will be represented in different ways when it's a free form string (mb/s vs MB/sec and mb vs MB vs MiB) and plurality. If this is only used for this package it should be ok maybe with a spot in the README or documentation that describes the different units of measure used. An enum could be even better but that may be too restrictive/overkill for this.

Reply:
Ah I see in the other PR you're using https://github.com/aws/aws-sdk-cpp/blob/master/aws-cpp-sdk-monitoring/include/aws/monitoring/model/StandardUnit.h for units. Perhaps that should be noted somewhere or used somehow in this package.

Reply:
I envision most of the consumers of this type of message will be designed to take in a free-form unit name.

However, Cloudwatch Metrics is not one of them, but it is within our control. `"percent"` does happen to be a compatible unit name: https://github.com/aws-robotics/cloudwatch-common/pull/48/files#diff-4a04cf878bd9a5f882f2d615a0017b03R33.

Reply:
Agree that we should not depend directly on CloudWatch units.